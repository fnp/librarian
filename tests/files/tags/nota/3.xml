<!-- albo też przed rozdziałami albo podrozdziałami, których dotyczą, lub po nich: -->

<akap>Jego marzenie się nie spełniło: nie ożenił się, choć był już na to zupełnie zdecydowany,
gdy skończono oporządzać jego mieszkanie. [...] Rzecz dziwna, czym jest mundur dla takich ludzi.</akap>

<nota><akap><wyroznienie>Uwaga tłumacza</wyroznienie>. Drukowane obecnie w ,,Nowej Reformie"
<tytul_dziela>Wspomnienia Sybiraka</tytul_dziela> (pamiętniki Józefa Bogusławskiego) pozwalają
szczęśliwym trafem podać w całości nazwiska Polaków, o których mówi Dostojewski w swoich pamiętnikach,
oznaczając te nazwiska tylko pierwszymi literami, z dodaniem czasami końcówek. [...]</akap></nota>
