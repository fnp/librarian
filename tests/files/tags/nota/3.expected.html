<div id="book-text">
<a href="#f1" class="wl-num">1</a><p class="wl paragraph" id="f1">Jego marzenie się nie spełniło: nie ożenił się, choć był już na to zupełnie zdecydowany,
gdy skończono oporządzać jego mieszkanie. [...] Rzecz dziwna, czym jest mundur dla takich ludzi.</p>
<div class="note"><p class="wl paragraph" id="i1"><em class="author-emphasis">Uwaga tłumacza</em>. Drukowane obecnie w „Nowej Reformie”
<em class="book-title">Wspomnienia Sybiraka</em> (pamiętniki Józefa Bogusławskiego) pozwalają
szczęśliwym trafem podać w całości nazwiska Polaków, o których mówi Dostojewski w swoich pamiętnikach,
oznaczając te nazwiska tylko pierwszymi literami, z dodaniem czasami końcówek. [...]</p></div>
</div>
